README.md

```diff
@@ -124,7 +125,7 @@ calculateWork :: Time Hour  -- type synonym for 'Time HourUnit'
               -> (Time WorkWeek, Time WorkDay)
 calculateWork workHours =
     let completeWeeks = floorUnit $ toUnit @WorkWeek workHours
-        completeDays  = floorUnit $ toUnit @WorkDay  workHours - toUnit completeWeeks
+        completeDays  = floorUnit $ toUnit @WorkDay  workHours -:- toUnit completeWeeks
```

Comment:
I guess second `toUnit` is redundant. Could you try to remove it and run this executable to see result?

Reply:
Then the types won't match (we want `completeDays` to be `Time WorkDay`)